File: PHP/Designer.php
<?php

require_once "Employee.php";
class Designer extends Employee
{
    private $type;

    public function __construct($id, $name, $lastName, $age,$type)
    {
        parent::__construct($id, $name, $lastName, $age);
        $this->type= $type;
    }


    public function getType()
    {
        return $this->type;
    }


    public function setType($type)
    {
        $this->type = $type;
    }


}

File: src/Designer.java
public class Designer extends Employee{

    private String type;

    public Designer(int id, String name, String lastName, int age,String type) {
        super(id, name, lastName, age);
        this.type=type;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    @Override
    public String toString() {
        return super.toString()+
                ", Designer in='" + type+"'";
    }
}
